src/configure/index.js

```diff
@@ -0,0 +1,34 @@
+const core = require('@actions/core');
+const io = require('@actions/io');
+const cache = require('@actions/cache');
+
+exports.getTelepresenceConfigPath = () => {
+    switch (process.platform) {
+        case "darwin":
+            return `/home/runner/Library/Application\\ Support/telepresence`;
+        case "linux":
+            return `/home/runner/.config/telepresence`;
```

Comment:
no string interpolation done here

```suggestion
            return '/home/runner/.config/telepresence';
```